src/page/home/styles.module.scss

```diff
@@ -0,0 +1,141 @@
+@use 'src/styles' as *;
+.container {
+    padding: 0px 100px;
+}
+
+.title {
+    margin: 70px 0px;
+}
+
+.video {
+    width: 70%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    border-radius: 24px;
+    margin-bottom: 14px;
+}
+
+.heading2 {
+    margin-bottom: 64px;
+    text-align: center;
+}
+
+.featureContainer {
+    margin: 140px 0;
+    .featureDetails {
+        @include flex-row;
+        >div {
+            width: 24%;
+        }
+        .icon {
+            width: 52px;
+            height: 52px;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+        .contentbox {
+            height: 80px;
+        }
+        .viewmore {
+            color: $color-primary;
+            text-decoration: none;
+        }
+    }
+}
+
+.solutionContainer {
+    @include flex-row;
+    margin-bottom: 140px;
+    .solutionContent {
+        width: 50%;
+        .solutionDetails {
+            padding: 22px;
+            .solutionIcon {
+                margin-right: 14px;
+            }
+        }
+    }
+    .imgStructure {
+        @include flex-row;
+        .imgLeft {
+            margin-right: 32px;
+            .imgFloatRight {
+                display: block;
+                margin-left: auto;
+            }
+        }
+        .imgRight {
+            margin-right: -100px;
+        }
+        .imgMarginLeft {
+            margin-left: 32px;
+        }
+        .imgRight20 {
+            margin-right: -20px;
+            height: 100%;
+        }
+    }
+}
+
+.userComments {
+    margin-bottom: 140px;
+    .comment {
+        @include flex-row;
+        flex-wrap: wrap;
+        >div {
+            width: 26%;
+            margin-bottom: 26px;
+        }
+    }
+}
+
+.newsContainer {
+    margin-bottom: 140px;
+    .news {
+        @include flex-row;
+        >div {
+            width: 29%;
+            padding-bottom: 24px;
+            img {
+                width: 100%;
+            }
+        }
+    }
+}
+
+.logos {
+    margin-bottom: 140px;
+    div {
+        display: flex;
+        justify-content: center
+    }
+}
+
+.contact {
+    @include flex-row;
+    margin-bottom: 140px;
+    .contactForm {
+        width: 45%;
+        form {
+            display: flex;
+            flex-direction: column;
+            margin: 40px 0;
+            .formInput {
+                padding: 16px;
+                color: $color-grey;
+                border: 1px solid $color-lightred;
+                border-radius: 10px;
+                 :focus {
+                    border: 1px solid $color-lightred;
+                    box-shadow: rgba(255, 255, 255, 255);
```

Comment:
rgba(red, green, blue, alpha)
alpha 0 - 1
this is $color-white for box-shadow ? pls check it again

Reply:
`textarea:focus,
input:focus {
outline: none;
box-shadow: 0px 8px 32px rgba(177, 177, 177, 0.5);}`
I mean this one. Forgot to remove it after testing it unsuccessfully.